main/define/test/src/mill/define/BasePathTests.scala

```diff
@@ -54,15 +65,6 @@ object BasePathTests extends TestSuite {
       checkMillSourcePath(TestGraphs.nestedCrosses)(_.cross("210").cross2("js"), "cross", "cross2")
     }
     test("overridden") {
-      object overriddenBasePath extends TestBaseModule {
-        override def millSourcePath = os.pwd / "overriddenBasePathRootValue"
-        object nested extends Module {
-          override def millSourcePath = super.millSourcePath / "overriddenBasePathNested"
-          object nested extends Module {
-            override def millSourcePath = super.millSourcePath / "overriddenBasePathDoubleNested"
-          }
-        }
-      }
```

Comment:
Was this move necessary? I wonder if it's incidental, or if it indicates a bug somewhere in utest/mill/scala3 that we should log somewhere